en/asset/model/glTF.md

```diff
@@ -0,0 +1,163 @@
+# glTF 模型
```

Comment:
Does this need to be English?

Reply:
Yes, when this pr is merged, I will add it to the list of pending translations for the issue #5638.